src/cache/messages/data/scalar/get.rs

```diff
@@ -152,30 +151,36 @@ impl<I: IntoBytes> From<I> for GetResponse {
 }
 
 /// Represents a value retrieved from the cache.
-#[derive(Debug, PartialEq, Eq)]
+#[derive(PartialEq, Eq, Default)]
 pub struct Value {
     /// The raw bytes of the item.
     pub(crate) raw_item: Vec<u8>,
 }
 
+impl std::fmt::Debug for Value {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        utils::write_bytes_for_debug(f, "raw_item", &self.raw_item)
+    }
+}
+
+impl std::fmt::Display for Value {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
```

Comment:
With this implementation, when I run the example and print the `GetResponse`, I get:

```bash
Listing caches:
my-test-cache
cache

Setting key: my_key, value: my_value
Got value: Ok(Hit { value: raw_item: [109, 121, 95, 118, 97, 108, 117, 101] (as string: "my_value") })
Got value: my_value
Permanently deleted cache named, cache
```

Reply:
I think that looks good! And the `<invalid UTF-8>` error case seems reasonable to me